Elena Krasnopolskaya/Order.js

```diff
@@ -0,0 +1,32 @@
+function Order(){
+    this.isComplete = false;
```

Comment:
Is's well working solution, but also you may use approaches of "freezing" such as Object.freeze() and Object.defineProperty(...,{writable: false}).